plugins/base/base-test-utils/src/main/kotlin/renderers/SignatureUtils.kt

```diff
@@ -11,7 +11,8 @@ fun TestOutputWriter.renderedContent(path: String = "root/example.html"): Elemen
         .single()
 
 fun Element.signature(): Elements = select("div.symbol.monospace")
-fun Element.firstSignature(): Element = signature().first()
+fun Element.firstSignature(): Element = signature().first() ?: throw IllegalStateException("No signature found")
```

Comment:
I suppose `NoSuchElementException` is better. See e.g.  `.first()` from stdlib

Reply:
Indeed, will fix